### PR TITLE
Add r rdimtools

### PR DESCRIPTION
**Pull request by TanguyGen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by TanguyGen** [comment]
@conda-forge/help-r  Thanks for reviewing !

**Comment by TanguyGen** [comment]
It seems it did not merged

**Comment by bgruening** [comment]
@TanguyGen the only explanation that I have is that this was because of the non-up-to-data branch here. You had not merged main into your branch. Maybe you can redo this PR?

**Comment by TanguyGen** [comment]
Ok thanks !

File: recipes/r-rdimtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    - r-rann
+    - r-rspectra
+    - r-rcpp >=0.12.15
+    - r-rcpparmadillo
+    - r-rcppde
+    - r-rcppdist
+    - r-rdpack
+    - r-maotai >=0.2.4
+    - r-mclustcomp
+    - m2w64-libwinpthread-git  # [win]
```
Comment (mfansler) [inline recipes/r-rdimtools/meta.yaml:75]: FYI, in the future, this should be expressed as:

```suggestion
    - {{ native }}libwinpthread-git  # [win]
```

**Pull request merged**